scipy/special/_convex_analysis.pxd

```diff
@@ -20,11 +21,26 @@ cdef inline double kl_div(double x, double y) noexcept nogil:
     else:
         return INFINITY
 
+@cython.cdivision(True)
 cdef inline double rel_entr(double x, double y) noexcept nogil:
+    cdef double ratio
+    # Smallest non-subnormal number, aka np.finfo(np.float64).tiny
+    cdef double tiny = 2.2250738585072014e-308
```

Comment:
You can get this value through

```cython
from libc.float cimport DBL_MIN
```

so no need to define it within the function.

Reply:
Thanks, thought there must be a better way to do this.

Fixed in 78b66f8.